manifests/init.pp

```diff
@@ -58,6 +58,10 @@
 #   or add options without having to recreate the entire hash. Defaults to
 #   false, but will default to true in future releases.
 #
+# @param install_options
+#   Array of arguments passed to the installer
+#   Default: []
```

Comment:
Remove the default, this is detected and automatically included in the generated doc.

```suggestion
```

Reply:
Removed